Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ViewTextController.java

```diff
@@ -42,7 +43,9 @@
         String respMessage = null;
         try {
             respMessage = tfManager.retrieveTextFileContent(txtId);
-            return new ResponseEntity<String>(respMessage, HttpStatus.OK);
+            String decodedRespMsg = StringEscapeUtils.unescapeXml(respMessage);
```

Comment:
Yea, changed code to store un escaped XML. But if the user sends un escaped xml via REST, it will still cause an exception.